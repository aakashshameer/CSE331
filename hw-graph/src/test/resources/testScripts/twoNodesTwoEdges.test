#Test to create two nodes, one self-edge and one edge between different nodes in the graph

#Create a graph
CreateGraph graphName

#Add two nodes
AddNode graphName b
AddNode graphName c

#Add one self edge
AddEdge graphName b b BB

#Add one edge between different nodes
AddEdge graphName b c BC

#Print the nodes in the graph
ListNode graphName

#Print outgoing edges from b
ListChildren graphName b

#Print outgoing edges from c
ListChildren graphName c