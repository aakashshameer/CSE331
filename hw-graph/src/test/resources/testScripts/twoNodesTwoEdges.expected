#Test to create two nodes, one self-edge and one edge between different nodes in the graph

#Create a graph
create graph dgraph

#Add two nodes
added node b to dgraph
added node c to dgraph

#Add one self edge
added edge BB from b to b in dgraph

#Add one edge between different nodes
added edge BC from b to c in dgraph

#Print the nodes in the graph
dgraph contains: b c

#Print outgoing edges from b
the children of b in dgraph are: b(BB) c(BC)

#Print outgoing edges from c
the children of c in dgraph are: